lib/readify.js

```diff
@@ -31,24 +31,77 @@ const nicki = !WIN && !BROWSER && require('nicki/legacy');
 const readdir = promisify(fs.readdir, fs);
 
 /* sorting on Win and node v0.8.0 */
-const sortFiles = sort((a, b) => {
-    return a.name > b.name ? 1 : -1;
+const sortFiles = currify((attr, order, array) => {
+    const cmpCallbacks = {
+        'numeric': (a, b) => (+a - +b),
+        'local_string': (a, b) => a.localeCompare(b.attr),
+        'default': (a, b) => (a > b ? 1 : -1),
```

Comment:
> localCompare is for accents. I'm french and I deal with accentuated letters like é è ë ê. 

That is interesting. Did not know about it.
Can you show me please how `ls -lha` sorts accents?